templates/web/base/report/update.html

```diff
@@ -43,9 +43,11 @@ <h4 class="static-with-rule">[% loc('Updates') %]</h4>
                     </div>
                     [% END %]
 
-                  [% SET update_state = update.problem_state_display(c) %]
-                  [% IF update_state AND update_state != global.last_state AND NOT (global.last_state == "" AND update.problem_state == 'confirmed') %]
-                    <p class="meta-2">[% loc('State changed to:') %] [% update_state %]</p>
+                  [% SET update_state = update.problem_state %]
+                  [% IF ( update_state AND update_state != global.last_state AND NOT (global.last_state == "" AND update.problem_state == 'confirmed') ) OR
+                     update.mark_fixed OR update.mark_open
```

Comment:
Hmm, so if there's an update that marks fixed but doesn't set problem state (227,000 of those), that'll set last_state to the empty string. I guess easiest thing to do is change the `SET update_state` so that it sets `update_state` to 'fixed' if mark_fixed, 'confirmed' if mark_open, problem_state otherwise. Think that should do it?